messenger/make.py

```diff
@@ -67,9 +76,55 @@ def build_octave():
     do_build(make_cmd, 'messenger.mex')
 
 
+def which_matlab():
```

Comment:
So the structure can be different.

I know that sometimes there is a `R2014_*` directory that is present in windows but not on OSX or Linux. I took care of that with this code.

Sorry, maybe I should have written more comments. Honestly didn't even think anyone would even read this :) 

```
        for p in os.listdir(os.path.join(programs[host], *matlab)):
            if p.startswith('R20'):
                matlab.append(p)

        matlab.append('bin')
```